src/datasources/tag/types.ts

```diff
@@ -12,20 +12,37 @@ export interface TagQuery extends DataQuery {
   properties: boolean;
 }
 
-export interface TagWithValue {
+interface TagWithValueBase {
   current: {
     value: { value: string };
     timestamp: string;
   } | null;
   tag: {
-    datatype: string;
     path: string;
     properties: Record<string, string> | null;
-    workspace: string;
-    workspace_id: string;
   };
 }
 
+// Legacy tag properties from SystemLink Server
```

Comment:
Is this comment accurate? It was my understanding that they changed to camelCase to match SLS.

Reply:
Shoot you're right. For some reason I was under the impression that it was the other way around. Probably wasn't worth writing the tests then.